user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -109,29 +115,34 @@ and change `/dev/sdY` to the correct target device e.g., `/dev/sdc`). Make sure
 to write to the entire device (e.g., `/dev/sdc`) rather than just a single
 partition (e.g., `/dev/sdc1`).
 
+#### Windows ISO to USB
+
 On Windows, you can use the [Rufus](https://rufus.akeo.ie/) tool to write the
-ISO to a USB key. MediaTest is not recommended. Be sure to select "DD image"
-mode (*after* selecting the Qubes ISO):
+ISO to a USB key. Be sure to select "Write in DD Image mode" *after* selecting
+the Qubes ISO and pressing "START" on the Rufus main window.
 
 <div class="alert alert-warning" role="alert">
   <i class="fa fa-exclamation-circle"></i>
-  <b>Note:</b>  If you do this on Windows 10, you can only install Qubes
-  without MediaTest, which is not recommended.
+  <b>Note:</b> Choosing this Windows+Rufus method means that you
+  [won't be able](https://github.com/QubesOS/qubes-issues/issues/2051) to choose
```

Comment:
Markdown syntax won't work here, because you're inside of an alert. Has to be HTML.